types/node-postal/index.d.ts

```diff
@@ -0,0 +1,51 @@
+// Type definitions for node-postal 1.1
+// Project: https://github.com/openvenues/node-postal
+// Definitions by: João Carmo <https://github.com/joaocarmo>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// Source: https://github.com/openvenues/libpostal#parser-labels
+export type PostalLabels =
+    | 'category'
+    | 'city_district'
+    | 'city'
+    | 'country_region'
+    | 'country'
+    | 'entrance'
+    | 'house_number'
+    | 'house'
+    | 'island'
+    | 'level'
+    | 'near'
+    | 'po_box'
+    | 'postcode'
+    | 'road'
+    | 'staircase'
+    | 'state_district'
+    | 'state'
+    | 'suburb'
+    | 'unit'
+    | 'world_region';
+
+export interface PostalResult {
+    component: PostalLabels;
+    value: string;
+}
+
+export function expand_address(address: string): string[];
+
+export function parse_address(address: string): PostalResult[];
+
+export const expand: {
+    expand_address: typeof expand_address;
+};
+
+export const parser: {
+    parse_address: typeof parse_address;
+};
+
+export const postal: {
+    expand: typeof expand;
+    parser: typeof parser;
+};
```

Comment:
those are internal symbols, the module exports only two symbols, nothing more IMO:
```suggestion
declare function expand_address(address: string): string[];

declare function parse_address(address: string): PostalResult[];

export const expand: {
    expand_address: typeof expand_address;
};

export const parser: {
    parse_address: typeof parse_address;
};

export {};
```
https://github.com/openvenues/node-postal/blob/master/index.js#L3-L4

Reply:
That is correct, no need to export extra stuff. Thank you for the `export {}` suggestion!